### DISCUSSION TITLE
Filter attachments

### DISCUSSION THREAD

**Discussion by loranger** [nocodb/nocodb#4890] [Q&A]
Hi,

Do you guys know how can I, from a dedicated view, filter all my records not having any attachment ?
It seems the filters *where attachment is empty* and *where attachment is null* does not apply on attachment fields

**Comment by dstala** [comment]
@loranger Looks a bug to me. Will discuss with team;
attachment cells, by default will hold `null` value. it should be `empty` after all attachments are removed.

**Comment by dstala** [comment]
You can try this as workaround, while we explore options to better serve this use case

![Screenshot 2023-01-20 at 1 31 07 PM](https://user-images.githubusercontent.com/86527202/213646028-3ccb5379-dda3-4c21-b422-d3f781bcb1c4.png)

**Reply by loranger** [reply]
Thanks for the workaround. Unfortunately it doesn't return a row, but that doesn't really matter: Knowing that you consider the feature is enough for me to wait 👍🏻
